roles/base/templates/docker_facts.j2

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+dver=$(sudo docker version | awk '/Version/{print $2; exit 0}')
+exper=$(sudo docker version | awk '/Experiment/{print $2; exit 0}')
```

Comment:
My only argument here is that we don't really need it to do anything we 
can't trivially do on the command line. If we expect a lot of change 
around this soon, then I will leave it alone, but the code 
footprint/orchestration is significant and I think we should measure 
carefully.

On 6 Dec 2015, at 15:04, jojimt wrote:

>> @@ -0,0 +1,20 @@
>> +#!/bin/sh
>> +
>> +dver=$(sudo docker version | awk '/Version/{print $2; exit 0}')
>> +exper=$(sudo docker version | awk '/Experiment/{print $2; exit 0}')
>
> With some changes to cover docker not being present etc., this can be 
> made to work. But, how is it better than what I implemented? I prefer 
> my method because it is easier to extend.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/3/files#r46777858